src/model/time-scale.ts

```diff
@@ -727,6 +731,18 @@ export class TimeScale {
 		return this._dateTimeFormatter.format(new Date(time.timestamp * 1000));
 	}
 
+	private _isAllScalingAndScrollingDisabled(): boolean {
+		return !this._model.options().handleScroll.horzTouchDrag
+			&& !this._model.options().handleScroll.mouseWheel
+			&& !this._model.options().handleScroll.pressedMouseMove
+			&& !this._model.options().handleScroll.vertTouchDrag
+			&& !this._model.options().handleScale.axisDoubleClickReset
+			&& !this._model.options().handleScale.axisPressedMouseMove.price
+			&& !this._model.options().handleScale.axisPressedMouseMove.time
+			&& !this._model.options().handleScale.mouseWheel
+			&& !this._model.options().handleScale.pinch;
```

Comment:
it seems that it might be easy to forget to add anything to this list. @kirchet maybe you might have an idea whether we can handle it differently.

```suggestion
		const { handleScroll, handleScale } = this._model.options();
		return !handleScroll.horzTouchDrag
			&& !handleScroll.mouseWheel
			&& !handleScroll.pressedMouseMove
			&& !handleScroll.vertTouchDrag
			&& !handleScale.axisDoubleClickReset
			&& !handleScale.axisPressedMouseMove.time
			&& !handleScale.mouseWheel
			&& !handleScale.pinch;
```

I'd change it to this but it doesn't change the situation a lot anyway (please note that I removed checking for `axisPressedMouseMove.price` since it seems that that's OK if you're able to change price range so why should we disable this feature in this case)

Reply:
> maybe you might have an idea whether we can handle it differently

I don't know, maybe we can add a helper that walks through all nestend properties and checks them for a provided predicate.

Reply:
I thought about that but it looks overcomplicated for me. Ok, let's apply the changes above and keep as is for now